src/config.rs

```diff
@@ -999,12 +999,21 @@ impl DbConfig {
         opts.set_log_file_time_to_roll(self.info_log_roll_time.as_secs());
         opts.set_keep_log_file_num(self.info_log_keep_log_file_num);
         if self.rate_bytes_per_sec.0 > 0 {
-            opts.set_ratelimiter_with_auto_tuned(
-                self.rate_bytes_per_sec.0 as i64,
-                (self.rate_limiter_refill_period.as_millis() * 1000) as i64,
-                self.rate_limiter_mode,
-                self.auto_tuned,
-            );
+            if self.auto_tuned {
```

Comment:
as we discussed before, shall we have this option default on, with `rate_bytes_per_sec` default to some large enough value (10G/s maybe?). or do we need more testing to confirm the new limier is effective?

Reply:
I think for now the test is not sufficient, but if default off it's hard to get feedback from extensive use cases. What's your advice?

Reply:
Also currently the limit is always greater than max/20 (like old implementation), so to default use 10G/s this needs to change.

Reply:
I think it is okay to default on in master, given the good result we are seeing.